c7n/resources/emr.py

```diff
@@ -75,6 +75,23 @@ def augment(self, resources):
 
 @actions.register('terminate')
 class Terminate(BaseAction):
+    """Action to terminate EMR cluster(s)
+
+    It is recommended to apply a filter to the terminate action to avoid
+    termination of all EMR clusters
+
+    :example:
+
+        .. code-block: yaml
+
+            policies:
+              - name: emr-terminate
+                resource: emr
```

Comment:
```
  - name: emr-clusters
    description:
        Report on any EMR clusters
    resource: emr
    query:
        - ClusterStates: [STARTING, BOOTSTRAPPING, RUNNING, WAITING]
```

please ensure that examples are tested, i'm not really clear which of these pull requests on doc examples have been fully vetted and which have not. fwiw, the one above has been.